.github/workflows/autobuild.yml

```diff
@@ -159,6 +265,6 @@ jobs:
         GITHUB_TOKEN:             ${{ secrets.GITHUB_TOKEN }}
       with:
         upload_url:               ${{ needs.create_release.outputs.upload_url }} # See this blog post for more info: https://jasonet.co/posts/new-features-of-github-actions/#passing-data-to-future-steps
-        asset_path:               /github/home/io.jamulus.Jamulus/io.jamulus.Jamulus.flatpak
-        asset_name:               io.jamulus.Jamulus.flatpak
+        asset_path:               ${{steps.flatpak-download.outputs.download-path}}/${{ steps.flatpak-prepare.outputs.flatpak_bundle }}
+        asset_name:               ${{ steps.flatpak-prepare.outputs.flatpak_name }}
```

Comment:
I'm not sure, but what happens on a PR? I saw that the release job ran. I hope, it doesn't automatically do a release on every PR. 

Another thought: would it be possible that this also moves the latest tag automatically if a tag starting with "r" is pushed? Unfortunately, I think we can't really move away from this.

Reply:
1) building artifacts is triggered on every pull request. Code QL is done. Artifacts are published as a part 
2) releases are triggered only if ((is_tag)  && (pattern "r_*" or "latest")) => not by requests, not by branches
Where is a good place to make the two above points clear? Where would you look for documentation of that? as a first suggestion, I will prepend the autobuild.yml with a few lines

3) if you plan to pop out releases every second day, I will implement automatisation for that happily. for releases every week or month, this gives you the opportunity to verify the builds as r_x_x_x. Moving "latest" needs you to move the tag only. one line of git command. I'd suggest that - let me know if you are really eager to have that step (that one git tag move&push) automated.

Reply:
1. Thanks for the clarification
2. I think that's enough
3. Unfortunately it's much more than one git command. You'll need to remove the latest tag locally, afterwards push the removed tag on GitHub, then re-add it locally and push it to github again. So it's much more than that unfortunately. I'd like to have this automated personally.

Reply:
@ann0see Is a release script already implemented? My suggestion would be to add those two commands to it:
git tag -f latest
git push --force github_nefarius latest;

Reply:
No. A release script is not yet implemented. I think you could use my proposed shell script in another issue.